src/app/app.component.ts

```diff
@@ -7,6 +7,8 @@ import { Component, OnInit } from '@angular/core';
 })
 export class AppComponent implements OnInit {
 
+    displayKeys: string[][] =  [ ["login", "Login"], ["name", "Name"], ["description", "Description"] ];
```

Comment:
Instead of an array of arrays, `displayKeys` could be an array of objects:
```suggestion
displayKeys: any[] = [ {"fieldName":"login","displayLabel":"Login"}, ... ]
```
It's more verbose but more readable. Not very important in this case but in the general case it does make a difference.
Better yet, instead of `any` could have defined a small interface with two string fields, _fieldName_ and _displayLabel_